boards/imix/src/main.rs

```diff
@@ -441,11 +509,9 @@ pub unsafe fn reset_handler() {
 
     let mut chip = sam4l::chip::Sam4l::new();
 
+    // These two lines need to be below the creation of the chip for
+    // initialization to work.
     rf233.reset();
-    rf233.set_pan(0xABCD);
-    rf233.set_address(0x1008);
-    //    rf233.config_commit();
-
     rf233.start();
```

Comment:
seems like `reset()` and `start()` ought to be in the Mac layer (or whichever component is the immediate client of `rf233`).

Reply:
So we're not entirely sure why, but the `reset()` and `start()` methods need to be below the creation of the chip otherwise the radio will not work, so I don't think they can be cleanly added to the current Mac initialization code. We could modify the Mac layer to expose a wrapper for `reset()` and `start()`, or we can try and figure out how to incorporate it more cleanly with the lower layers.

In either case, I'm not sure if we should implement it in this pull request, or submit another (small) pull request that modifies the Mac layer.